### ISSUE TITLE
Error WPS428 for ellipsis in @overload functions

### ISSUE THREAD

**Issue by vnmabus** [wemake-services/wemake-python-styleguide#1777]
# Bug report

## What's wrong

Functions decorated with the `@overload` decorator of the `typing` module should have an empty body, usually denoted with ellipsis, as in the default example:

```python
@overload
def process(response: None) -> None:
    ...
```

However, WPS reports this usage as `WPS428 Found statement that has no effect`.

## How it should be

Ellipsis should be accepted as the body of `@overload` functions.

**Comment by sobolevn** [comment]
Yes, we prefer adding docstrings instead of `...` in each overload. So, this allow us to tell why whould you need each of this cases.

Example: https://github.com/dry-python/returns/blob/master/returns/methods/cond.py#L20

**Comment by vnmabus** [comment]
> 
> 
> Yes, we prefer adding docstrings instead of `...` in each overload. So, this allow us to tell why whould you need each of this cases.
> 
> Example: https://github.com/dry-python/returns/blob/master/returns/methods/cond.py#L20

Sorry, but this goes against standard practice, and the next version of pydocstyle will even flag it as an error D418 (see https://github.com/PyCQA/pydocstyle/pull/511).

**Comment by sobolevn** [comment]
Thanks! I was not aware about this new rule in pydocstyle! We would need to swith off its rule for that.
I really believe that this solution is more readable than `...`.
